src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -165,11 +161,11 @@ private void openBrowserImpl(String uri, String page) {
   public static void startServer(
     Project project,
     FlutterSdk sdk,
-    PubRoot pubRoot,
     Callback<DevToolsInstance> onSuccess,
     Callback<DevToolsInstance> onClose
   ) {
-    final FlutterCommand command = sdk.flutterPackagesPub(pubRoot, "global", "run", "devtools", "--machine", "--port=0");
+    // We don't need a pubroot to call pub global.
```

Comment:
good catch

Reply:
Interestingly, on paper we don't need it, but the pubspec.yaml check in the Flutter tool is too aggressive:

https://github.com/flutter/flutter/issues/33324